addon/mixins/imgix-path-behavior.js

```diff
@@ -68,70 +76,83 @@ export default Ember.Mixin.create({
    * @property {string}
    * @return the fully built string
    */
-  src: computed('_path', '_query', '_width', '_height', '_dpr', 'crop', 'fit', function () {
-    if (!this.get('_width')) { return; }
-
-    let env = this.get('_config');
-
-    // These operations are defaults and should be overidden by any incoming
-    // query parameters
-    let options = {
-      crop: this.get('crop') || "faces",
-      fit: this.get('fit') || "crop"
-    };
-
-    if (this.get('auto')) {
-      merge(options, { auto: this.get('auto') });
+  src: computed(
+    '_path',
+    '_query',
+    '_width',
+    '_height',
+    '_dpr',
+    'crop',
+    'fit',
+    function() {
+      if (!get(this, '_width')) {
+        return;
+      }
+
+      let env = get(this, '_config');
+
+      // These operations are defaults and should be overidden by any incoming
+      // query parameters
+      let options = {
+        crop: get(this, 'crop') || 'faces',
```

Comment:
I don't think this is the best default behavior here. I agree that `fit=crop` is the best default fitting algorithm to use, but I think cropping to faces by default will potentially be a source of confusion for developers. I'd rather leave `crop` off entirely, which will default to centering the crop rectangle.

Reply:
As this already exists in the library before this PR, I'm going to pull this out into a PR to fix: #34